src/cloud/project/new-relic.md

```diff
@@ -268,55 +274,77 @@ Prerequisites
 
-   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-alert-policies.png){:width="650px"}
+1. You should see the _Managed Alerts for Magento Commerce_ policy.
+
+   ![Generated alert policies]({{ site.baseurl }}/common/images/cloud/cloud-new-relic-magento-managed-alerts.png){:width="650px"}
 
    {:.bs-callout-info}
-   If you do not see these alert policies, submit a Magento support ticket. Include your project ID in the ticket. These policies are available on Pro and Starter Production environments only.
+   If you do not see this alert policy, submit a Magento support ticket. Include your project ID in the ticket. The policy is available on Pro and Starter Production environments only.
+
+1. Click **Managed Alert policies for Magento Commerce** to review the alert conditions defined in the policy.
 
-1. Click an alert policy name to review the threshold conditions that trigger alerts.
+   ![Managed alerts list]({{ site.baseurl }}/common/images/cloud/cloud-newrelic-managed-alerts-for-magento.png){:width="650px"}
 
 ### Configure notification channels
 
-To use the Adobe-generated alert policies to monitor your Production sites, you must configure notification channels and map them to alert policies. Notifications about performance issues go to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.
+To use the Managed alerts for Magento Commerce policy to monitor your Production sites, you must configure notification channels and map them to alert policies.
 
-New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels] in the New Relic documentation to review the prerequisites for using each type.
+Notifications about performance issues go to all channels associated with an alert policy when conditions on the application or infrastructure trigger an alert. You also receive notifications when an issue is acknowledged and closed.
 
-The following instructions describe the high level steps to configure a notification channel to receive alert messages triggered by an alert policy.
+New Relic provides templates for configuring different types of notification channels including email, Slack, PagerDuty, webhooks, and more. See the [Instructions for specific notification channels] in the New Relic documentation to review the prerequisites for using each type.
 
 {:.procedure}
 To configure a notification channel:
 
 1. Choose a notification channel type, and complete any [prerequisite steps] required to connect the channel with the New Relic service.
 
-1. [Log in to your New Relic account][New Relic login].
+1. Use your New Relic credentials to [log in to your New Relic account][New Relic login].
+
+1. Open your application.
```

Comment:
Again, this step was not required for me. It may be like you said, because of our insider's viewpoint. But the Applied Intelligence page has a drop-down to select your project or account.
<img width="504" alt="Screen Shot 2020-09-01 at 11 11 19" src="https://user-images.githubusercontent.com/5817319/91877692-f5dbad00-ec43-11ea-8bbe-89694ecf4c20.png">